examples/large_models/Huggingface_pippy/pippy_handler.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import time
+from abc import ABC
+
+import requests
+import torch
+import transformers
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ts.handler_utils.distributed.pt_pippy import get_pipeline_driver
+from ts.torch_handler.distributed.base_pippy_handler import BasePippyHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+torch.manual_seed(40)
```

Comment:
Not sure how I feel about this? What if users want another seed. Make it an explicit config if its important enough